<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrokerRelaStatePop</class>
 <widget class="QDialog" name="BrokerRelaStatePop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>区间设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,2,2,2,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>区间阶段编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="la_num">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>区间起始值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_start"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>区间结束值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_end"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>提成比例</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ratio"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancle_btn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>若要设置固定比例方案，则区间编号默认为1且不可更改，区间起始值设为0，结束值设为0即可</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>若要设置分阶段比例方案，最后一个区间结束值设为0，阶段编号默认递增，删除中间区间会导致编号产生断层，会影响最终的保存操作，但可以在上一个区间处点击增加区间保证编号正常</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
